src/ADAL.PCL/TokenCache.cs

```diff
@@ -344,13 +344,16 @@ internal AuthenticationResultEx LoadFromCache(CacheQueryData cacheQueryData, Cal
         {
             lock (cacheLock)
             {
+                Telemetry.GetInstance().StartEvent(callState.RequestId, "cache_lookup");
+                CacheEvent cacheEvent = new CacheEvent();
```

Comment:
The event name is accounted for in the start event.

---
In reply to: [83751389](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r83751389) [](ancestors = 83751389)